src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordToExistingCaseTest.kt

```diff
@@ -31,45 +40,92 @@ fun RequestSpecification.postToCallback(type: String = "attach_case") = post("/c
 fun RequestSpecification.setBody(builder: CallbackRequestBuilder) = body(builder.build())
 fun ResponseValidation.shouldContainError(error: String) = body("errors", hasItem(error))
 
+fun MappingBuilder.authorised() = with(this) {
+    withHeader(AUTHORIZATION, containing("eyJhbGciOiJIUzI1NiJ9."))
+    //TODO cant seem to make this match
+//    withHeader("ServiceAuthorization", containing("eyJhbGciOiJIUzI1NiJ9."))
```

Comment:
can those comments be removed?

Reply:
well ... its supposed to have both so we should match on both but it seems to not work at this point.